syft/internal/packagemetadata/names.go

```diff
@@ -109,6 +109,7 @@ var jsonTypes = makeJSONTypes(
 	jsonNames(pkg.RustCargoLockEntry{}, "rust-cargo-lock-entry", "RustCargoPackageMetadata"),
 	jsonNamesWithoutLookup(pkg.RustBinaryAuditEntry{}, "rust-cargo-audit-entry", "RustCargoPackageMetadata"), // the legacy value is split into two types, where the other is preferred
 	jsonNames(pkg.WordpressPluginEntry{}, "wordpress-plugin-entry", "WordpressMetadata"),
+	jsonNames(pkg.HomebrewMetadata{}, "homebrew-metadata"),
```

Comment:
I believe instead of metadata, we would use entry, so `HomebrewEntry` and `homebrew-entry`

Reply:
Since it's not an entry from multiple entries within a file I kept to ecosystem specific verbiage: `HomebrewFormula`